pydrive2/test/.gitignore

```diff
@@ -0,0 +1,6 @@
+configs/*
+credentials/*
+settings/local/*
+
+client_secrets.json
```

Comment:
is is supposed to be in configs? may be we indeed can make it tests/client_secrets.json always? to avoid configs/settings naming confusion

Reply:
sounds good! `configs` directory is removed and `client_sercrets.json` should reside in `pydrive2/test`.